packages/console/src/components/FormCard/index.module.scss

```diff
@@ -30,6 +30,14 @@
   }
 }
 
+
+@media screen and (max-width: 1080px) {
+  .introduction {
+    display: none;
+  }
+}
```

Comment:
According to the Figma design, we should display the description on top of the form fields if the width is less than 1080px

Reply:
Updated.